### PR TITLE
Trade UI cleanup

### PR DESCRIPTION
**Pull request by 1saf**
# Description

Adds in a few quality of life changes to the trade UI.
1. People have complained about the price graph being 'inverse' so, it's been inversed to un-inverse the way the users would see it.
2. Add in a small icon tooltip to indicate where the pricing data for the graph is coming from.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Other

## How should this be tested?

Ensure that the pair price graph is showing the pricing data in terms of `outputAsset/inputAsset` and in label too
Ensure that there is a small coingecko icon next to the pair symbols on the price graph and has a tooltip

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code where relevant, particularly in hard-to-understand areas
- [X] My changes generate no new console warnings
- [X] The base of this PR is `master` if hotfix, `develop` if not

## Walkthrough
https://www.loom.com/share/b43bbcdc2fbf45c1831ae701f485581d

### CODE REVIEW

File: src/components/cards/PairPriceGraph/PairPriceGraph.vue [hunk 1]
```diff
@@ -21,17 +21,33 @@ import { getAddress } from '@ethersproject/address';
...
+  days: number,
+  inverse?: boolean
 ) {
   let _inputAsset =
     inputAsset === nativeAsset ? wrappedNativeAsset : inputAsset;
   let _outputAsset =
     outputAsset === nativeAsset ? wrappedNativeAsset : outputAsset;
+
+  if (inverse) {
+    const tempInputAsset = _inputAsset;
```
Comment (evgenyboxer) [inline src/components/cards/PairPriceGraph/PairPriceGraph.vue:47]: could you try `[_inputAsset, _outputAsset] = [_outputAsset, _inputAsset];`

**Review by evgenyboxer — APPROVED** [review]
lgtm!

Two suggestions:
1. Perhaps allow the tooltip to big a bit bigger to accomodate for "Coingecko" (takes 3 lines right now, perhaps would look better with 2 lines)
2. Link out to the pair on Coingecko? the cursor looks as if its clickable.

File: src/components/cards/PairPriceGraph/PairPriceGraph.vue (hunk 1 above)
Comment (1saf) [reply src/components/cards/PairPriceGraph/PairPriceGraph.vue:47]: oh wow, didn't know you could do that

**Comment by 1saf** [comment]
> 

I don't think there is a pair view on coingecko

**Review by timjrobinson — APPROVED** [review]
LGTM. I also think the graph looks more correct this way.

**Pull request merged**